Task72/Task72/Program.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task72
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите x и y");
+
+            int x = int.Parse(Console.ReadLine());
+            int y = int.Parse(Console.ReadLine());
+            Console.WriteLine(TrueFalse(x, y));
+            Console.ReadKey();  
+
+        }
+        private static bool TrueFalse(int x, int y)
+        {
+            if ((x >= 2) && (y >= 0))
+                return true;
+            else if ((x >= 1) && (y <= -1))
+                return true;
+            else
+                return false;
+
+        }
```

Comment:
Методы класса лучше разделять пустой строкой. Так код читается легче. А вот перед `}` пустая строка лишняя.
Имя метода невнятное. Из него сложно понять, **что** метод делает.
Координаты точки не обязательно целые.
Не нужно заключать в скобки неравенства. Это  затрудняет чтение кода. Неравенства имеют более высокий приоритет, чем логические операции.
Если внутри `if` стоит `return`, то можно обойтись без `else`.
```suggestion
        }
        
        private static bool IstInArea(double x, double y)
        {
            if (x >= 2 && y >= 0)
                return true;
                
            if (x >= 1 && y <= -1)
                return true;
            
            return false;
        }
```

А ещё лучше было использовать только одно логическое выражение и всего один оператор `return`.
```suggestion
        }
        private static bool IstInArea(double x, double y)
        {
            return (x >= 2 && y >= 0) || (x >= 1 && y <= -1);
        }
```